WcaOnRails/db/migrate/20170503205810_add_index_on_results_country_id.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddIndexOnResultsCountryId < ActiveRecord::Migration[5.0]
+  def change
+    add_index :Results, :countryId
```

Comment:
This speeds up [this](https://github.com/thewca/worldcubeassociation.org/pull/1591/files#diff-690a237f897949397fe138a647116656R145) query.